snapshot/pkg/apis/crd/v1/types.go

```diff
@@ -235,6 +241,9 @@ type VolumeSnapshotDataSource struct {
 	// kubelet's host machine and then exposed to the pod.
 	// More info: https://kubernetes.io/docs/concepts/storage/volumes#awselasticblockstore
 	// +optional
+	//GlusterSnapshotSource represents a gluster snapshot resource
+	GlusterSnapshotVolume *GlusterVolumeSnapshotSource `json:"glusterSnapshotVolume,omitempty"`
```

Comment:
It's not discoverable by the controller: you should also update the GetSupportedVolumeFromPVSpec and GetSupportedVolumeFromSnapshotDataSpec functions.

Reply:
@tsmetana Thanks, Done. :)